src/70_User_Consent/index.json

```diff
@@ -0,0 +1,3 @@
+{
+  "description": "Samples demonstrating how to implement ad and analytics integrations in AMP."
```

Comment:
Is this true? Isn't this to demonstrate the user consent?

Reply:
yes, forgot to change it. done.